### ISSUE TITLE
[SUGGESTION] Alt+left/right arrow in Editor to go through cursor positions history.

### ISSUE THREAD

**Issue by paweljarosz** [defold/editor2-issues#2594]
As discussed here: https://forum.defold.com/t/alt-arrow-left-right-to-get-to-the-previous-next-cursor-position-in-editor/51721

I think it would be nice for Alt + left/right key shortcut to have a functionality that you can smartly get back to your last cursor position just like in VS Code. 
It omits last cursor position changes, when they were made using arrows keys in the current file, but skips to the last position in other file or last clicked position by mouse if it was in the part not currently visible in the window. 
When you are working with many scripts/modules it's sometimes quicker to browse through the history like this to get back to the particular part of the code you were editing in a blink of an eye, especially needed when you are working in a huge script and works in two or more different places of it, for example editing something in init() at the beginning and in on_message() near the end.

**Comment by matgis-king** [comment]
Agreed. I use this a lot. It doesn't seem like the standard shortcut for this is <kbd>Alt</kbd> + <kbd>Left</kbd> or <kbd>Right</kbd>, though.

**Comment by paweljarosz** [comment]
It works in every MS product I use. It works in browsers. For me it is very intuitive.
Urban dictionary even describes it :D https://www.urbandictionary.com/define.php?term=Alt-Left

**Comment by nikolajevs86** [comment]
Or CMD + Left/Right on Macs, like in IntelJ :)
Is this feature planned to implement?

**Comment by matgis-king** [comment]
I personally want this feature as well. We hope to be able to support it eventually.
